tests/_support/utils/tests/testUtil-test.js

```diff
@@ -0,0 +1,16 @@
+// const testUtil = require('../testUtil');
```

Comment:
excellent, I knew they wanted to make it possible to run unit tests in Cypress, but didn't know this is already possible.